docs/requirements.txt

```diff
@@ -15,6 +15,7 @@ sphinx-rtd-theme
 sphinx-autodoc-typehints
 sphinx-inline-tabs
 sphinxcontrib-bibtex
+sphinxcontrib.jquery
```

Comment:
please remove this, it is implicitly required by rtd theme :)

Reply:
See here: https://github.com/readthedocs/sphinx_rtd_theme/issues/1452#issuecomment-1490504991

Reply:
Hmm I ran `pip install -r requirements.txt` and it still told me that this package was missing. Perhaps it is only from a certain version of the RTD theme?

Reply:
Hmm, lets remove it as that is recommended by the rtd theme. Hmm.. Not that many are going to do it anyways, it is more if rtd decides to use another backend for search. gh-actions worked without problems, so perhaps something on your end?

Reply:
Surely, I think it is just that I had an old version of the RTD theme already installed, that's why probably in the requirements the version of the RTD theme should be specified